packages/x-components/src/x-modules/facets/events.types.ts

```diff
@@ -57,7 +58,7 @@ export interface FacetsXEvents {
    * @remarks This event does not imply changing the selection state of the filter. Business logic
    * can prevent the filter from changing its state.
    */
-  UserClickedAFilter: Filter;
+  UserClickedAFilter: Filter | FilterNext;
```

Comment:
I think we can leave here the `Filter` right? 
we are using the `UserClickedANextFilter` event in the new module

Reply:
Yep, this was from a previous approach and I forgot to remove it.